### PR TITLE
Openshift origin rhel

### PR DESCRIPTION
**Pull request by haroldwongms**
### Contributing guide
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

### Changelog
README.md
azuredeploy.json
azuredeploy.parameters.json

### Description of the change
README.md: updated title to reflect username / password, corrected instructions for deployment
azuredeploy.json: updated method for incorporating keyvault, included option to select CentOS or RHEL
azuredeploy.parameters.json: updated to include new parameters

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@haroldwongms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (haroldw). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by BorisB2015** [comment]
@gatneil - please take a look. Thanks!

File: openshift-origin-rhel/azuredeploy.json [hunk 1]
```diff
@@ -2,19 +2,27 @@
 	"$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
 	"contentVersion": "1.0.0.0",
 	"parameters": {
+		"baseUrl": {
```
Comment (gatneil) [inline openshift-origin-rhel/azuredeploy.json]: please see the best guidance on _artifactsLocation and _artifactsLocationSasToken: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

File: openshift-origin-rhel/nested/openshiftdeploy.json [hunk 2]
```diff
@@ -2,6 +2,12 @@
 	"$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
 	"contentVersion": "1.0.0.0",
 	"parameters": {
+		"baseUrl": {
```
Comment (gatneil) [inline openshift-origin-rhel/nested/openshiftdeploy.json]: _artifactsLocation and _artifactsLocationSasToken

**Comment by gatneil** [comment]
Merging for now, but please address the above comments post-merge

**Pull request merged**